WasatchVCPPLib/WasatchVCPPLib/WasatchVCPPWrapper.cpp

```diff
@@ -594,6 +594,35 @@ int wp_get_max_timeout_ms(int specIndex)
     return spec->maxTimeoutMS;
 }
 
+int wp_write_eeprom_page(int specIndex, int pageIndex, unsigned char* data, int dataLen)
+{
+    auto spec = driver->getSpectrometer(specIndex);
+    if (spec == nullptr)
+        return WP_ERROR_INVALID_SPECTROMETER;
+
+    if (pageIndex < 0)
+    {
+        return WP_ERROR;
+    }
+    if (dataLen != 64)
+    {
+        return WP_ERROR;
+    }
+
+    bool specType = spec->isARM();
+    if (specType) 
+    {
+        wp_send_control_msg(specIndex, 0xff, 0x02, 4, data, dataLen);
+    }
+    else
+    {
+        unsigned int pageValue = (0x3c << 8) | (0x40 * pageIndex);
+        wp_send_control_msg(specIndex, 0xa2, pageValue, 0, data, dataLen);
+    }
+    
+    return 0;

```

Comment:
not a bug, but minor comment: instead of returning 0, you could return whatever came back from the calls to wp_send_control_msg above (would allow them to flow-back errors on timeouts or whatever).